src/libs/database.ts

```diff
@@ -0,0 +1,43 @@
+import 'reflect-metadata';
+import { Container, Service } from 'typedi';
+import { Connection, createConnection, useContainer } from 'typeorm';
+
+@Service()
+export class Database {
+
+  private connection: Connection;
+
+  public async connect(): Promise<void> {
+    if (!this.connection) { await this.bootstrap(); }
+    if (!this.connection.isConnected) { await this.connection.connect(); }
+  }
+
+  public async disconnect(): Promise<void> {
+    if (this.connection && this.connection.isConnected) { await this.connection.close(); }
+  }
+
+  private getConfig() {
```

Comment:
I think we can simplify this with the current version in getting the `ormconfig` because if we add another environment config, we will need to adjust this one again. We already have the naming convention of `ormconfig.${NODE_ENV}.json` - which I think is already straight-forward for a developer to understand.

Reply:
yeah I agree on this one somehow, but I am wondering if the developer is trying to test the app we are giving them a chance to run a test that uses a different ormconfig. Somehow not good but not all the time

Reply:
The developer can change the credentials instead of providing long lines of code that can be simplified